### PR TITLE
Parse host_name for send_logs_level

### PR DESCRIPTION
**Pull request by azat**

### CODE REVIEW

**Comment by xzkostyan** [comment]
Makes sense since we have `alt_hosts` parameter.

**Comment by azat** [comment]
>Makes sense since we have alt_hosts parameter.

And not only `alt_hosts` but also for distributed queries server (entry point) will proxy logs from all servers that was queried (shards)

File: clickhouse_driver/log.py [hunk 1]
```diff
@@ -30,9 +30,14 @@ def log_block(block):
         # thread_number in servers prior 20.x
         thread_id = row.get('thread_id') or row['thread_number']
 
+        host_name = row.get('host_name')
+        if host_name:
+            host_name = '[ {} ] '.format(host_name)
+
         logger.info(
-            '{%s} [ %s ] <%s> %s: %s',
+            '{%s} %s[ %s ] <%s> %s: %s',
```
Comment (xzkostyan) [inline clickhouse_driver/log.py:38]: `host_name` is always present in the `row`. We can use it directly in `logger.info`:

```python
'{%s} [ %s ][ %s ] <%s> %s: %s',
...
row['host_name']
```

Comment (azat) [reply clickhouse_driver/log.py:38]: Yeah, I also dislike this extra concatenation, but I saw the same in the clickhouse sources, I guess that it was not always presented in the block:

https://github.com/ClickHouse/ClickHouse/blob/13d22bcd422f65141a5ddea7a75d1bd332b054c5/src/DataStreams/InternalTextLogsRowOutputStream.cpp#L37

Added in a0e9d17bfdfa4a65f14de2a9c885a438c1c43b92 (v1.1.54386-testing-178-ga0e9d17bfd)

Comment (azat) [reply clickhouse_driver/log.py:38]: (I though about adding a comment but by some reason prefer not to, sigh)

Comment (xzkostyan) [reply clickhouse_driver/log.py:38]: Okay. It seems that `query_id` is [also optional](https://github.com/ClickHouse/ClickHouse/blob/13d22bcd422f65141a5ddea7a75d1bd332b054c5/src/DataStreams/InternalTextLogsRowOutputStream.cpp#L69).

We have two options to keep things consistent here:
* to make `query_id` optional;
* to make `host_name` not optional.

AFAIK `query_id` always should be present. I think empty strings handling is done for pretty formatting in ClickHouse server's sources.

Let's try to make `host_name` not optional. What do you think?

Comment (azat) [reply clickhouse_driver/log.py:38]: > Let's try to make host_name not optional. 

Agree

File: clickhouse_driver/log.py [hunk 2]
```diff
@@ -31,8 +31,9 @@ def log_block(block):
         thread_id = row.get('thread_id') or row['thread_number']
 
         logger.info(
-            '{%s} [ %s ] <%s> %s: %s',
+            '{%s} [ %s ] [ %s ] <%s> %s: %s',
             row['query_id'],
+            row['host_name'],
```
Comment (xzkostyan) [inline clickhouse_driver/log.py:36]: Good. But it seems that `host_name` [should be before](https://github.com/ClickHouse/ClickHouse/blob/13d22bcd422f65141a5ddea7a75d1bd332b054c5/src/DataStreams/InternalTextLogsRowOutputStream.cpp#L37) `query_id`.

Comment (azat) [reply clickhouse_driver/log.py:36]: So as thread_id

**Pull request merged**